hil-test/Cargo.toml

```
@@ -28,31 +28,37 @@ required-features = ["async", "embassy"]
 [dependencies]
 defmt              = { version = "0.3.6" }
 defmt-rtt          = { version = "0.4.0" }
-embassy-time       = { version = "0.3.0", features = ["generic-queue-8"] }
+embassy-time       = { version = "0.3.0", features = ["generic-queue-64"] }
+embassy-futures    = { version = "0.1" }
 embedded-hal       = { version = "1.0.0" }
 embedded-hal-02    = { version = "0.2.7", package = "embedded-hal", features = ["unproven"] }
 embedded-hal-async = { version = "1.0.0", optional = true }
 embedded-hal-nb    = { version = "1.0.0", optional = true }
-esp-hal            = { path = "../esp-hal", features = ["defmt", "embedded-hal", "embedded-hal-02"], optional = true }
-semihosting        = { git = "https://github.com/taiki-e/semihosting", rev = "c829c19" }
+esp-hal            = { path = "../esp-hal", features = ["defmt", "embedded-hal", "embedded-hal-02", "defmt"], optional = true }
```

```suggestion
esp-hal            = { path = "../esp-hal", features = ["defmt", "embedded-hal", "embedded-hal-02"], optional = true }
```

`defmt` feature is duplicated here